internal/lightcone/abundance/timewaitsfornoone/timewaitsfornoone.go

```diff
@@ -0,0 +1,95 @@
+package timewaitsfornoone
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/lightcone"
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	time = "time-waits-for-no-one"
+	cd   = "time-waits-for-no-one-extra-dmg-cd"
+)
+
+// Desc : Increases the wearer's Max HP by 18% and Outgoing Healing by 12%.
+// When the wearer heals allies, record the amount of Outgoing Healing.
+// When any ally launches an attack, a random attacked enemy takes Additional DMG
+// equal to 36% of the recorded Outgoing Healing value.
+// The type of this Additional DMG is of the same Type as the wearer's.
+// This Additional DMG is not affected by other buffs, and can only occur 1 time per turn.
+
+func init() {
+	lightcone.Register(key.TimeWaitsforNoOne, lightcone.Config{
+		CreatePassive: Create,
+		Rarity:        5,
+		Path:          model.Path_ABUNDANCE,
+		Promotions:    promotions,
+	})
+	modifier.Register(time, modifier.Config{})
+	// added to automatically track extraDmg cooldown
+	modifier.Register(cd, modifier.Config{})
```

Comment:
CD needs to have its `TickMoment` as `ModifierPhase1End` so it disables at the start of the turn and not the end (which is what it currently does with the  default `ModifierPhase2End` tick moment)